config/environments/development.rb

```diff
@@ -87,8 +100,12 @@
     config.cache_classes = true
     config.eager_load = true
 
-    config.logger = ActiveSupport::Logger.new($stdout)
     config.log_level = :info
+    config.rails_semantic_logger.format     = :json
+    config.rails_semantic_logger.semantic   = true
+    config.rails_semantic_logger.started    = false
+    config.rails_semantic_logger.processing = false
+    config.rails_semantic_logger.rendered   = false
```

Comment:
am I misreading this, or does this override the section above and force it back to JSON regardless of the ENV var?

Reply:
only if `ENV["PROFILE"]` is set